### ISSUE TITLE
Demo not working

### ISSUE THREAD

**Issue by christophfriedrich** [spacebel/geozarr-openlayers#2]
Hi, I stumbled across this and it looks quite interesting, so I'm very curious to try it! The readme links to plenty of demo _videos_, but is there also a demo _instance_ hosted online where I can just interactively try it myself?

I thought I could just do-it-myself by cloning the repository and opening the index.html locally in my browser. But even after installing an add-on that bypasses the CORS issues, I was still not able to see anything as the server from where the data is requested responds with 401 Unauthorized...

**Comment by christophenoel** [comment]
HI @christophfriedrich . Unfortunately, the data is not available anymore.

I don't know when I'll find time to restore some data, but please refer to the repository: https://github.com/zarr-developers/geozarr-spec/
(note that the conventions are under development)

**Comment by christophfriedrich** [comment]
That's what I was fearing... It would be nice to add the demo data to this repository so that it's independent.

Thanks for the reference -- I'm aware of the spec repository, but was much more interested in an actual implementation, especially with OpenLayers.

**Comment by christophenoel** [comment]
Yes. The data was kind of "copyrighted" by another company, and the project is completed for a long time now.
All apologies for that.

**Comment by christophfriedrich** [comment]
I understand :/ Maybe it's possible to adapt the demo to some dummy data that may not look as interesting but proves the technical point. But I know doing stuff for finished projects is not a priority... Thanks for your replies!
